<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin::DeviceViewWidget</class>
 <widget class="QWidget" name="Admin::DeviceViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1427</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QToolButton" name="DVW_btn_show_hide">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="CustomTreeView" name="DVW_tree_item"/>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QMdiArea" name="mdiArea"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTreeView</class>
   <extends>QTreeView</extends>
   <header>app/customtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
